lib/discourse_chat/provider/slack/slack_message.rb

```diff
@@ -36,8 +37,14 @@ def text
         text = parts.length > 1 ? parts[1] : parts[0]
 
         if parts[0].start_with?('@')
-          user = @transcript.users.find { |u| u["id"] == parts[0].gsub('@', '') }
-          next "@#{user['name']}"
+          user_id = parts[0].gsub('@', '')
```

Comment:
The `@` would only appear at the beginning of the string right? Rather than going through the entire string you could do something like `user_id = parts[0][1..-1] if parts[0].start_with?('@')`

Reply:
We already checked `if parts[0].start_with?('@')` above, so we can just use `user_id = parts[0][1..-1]`.